templates/webapi/not_found.html.ep

```diff
@@ -23,7 +23,7 @@ table pre {
             % $pattern = "+$pattern" if $depth;
             <pre><%= '  ' x $depth %><%= $pattern %></pre>
         </td>
-        <td><%= uc(join ',', @{$route->via || []}) || '*' %></td>
+        <td><%= uc(join ',', @{$route->methods || []}) || '*' %></td>
```

Comment:
this fails tests because the old Mojo version does not have this method yet. Instead of breaking support for older Mojo versions immediately I suggest to fallback to `via` if `methods` is not defined and only bump the version in `.circleci/ci-packages.txt`, not introduce a version requirement.